src/docs/teams/account/redeeming-a-coupon.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Teams | Redeeming a coupon
+layout: docs.hbs
+---
+
+# Redeeming a coupon
+
+Redeeming a coupon for a Truffle Teams paid subscription is different for new and existing subscriptions. Please be sure to read the instructions carefully to make sure your coupon is applied correctly. If at any point you need some help, please don't hesitate to [ask us a question](/docs/teams/support/ask-us-a-question).
+
+- [New Subscriptions](#new-subscriptions)
+- [Existing Subscriptions](#existing-subscriptions)
+
+## New Subscriptions
+
+Before you can continue below, you'll need to make sure you have [created an account](/docs/teams/getting-started/creating-an-account) **and** [added at least one repository](/docs/teams/getting-started/adding-repositories) for the GitHub user/organization you'd like to apply your coupon code for.
+
+Once you've made sure the account you'd like to apply the coupon for exists, navigate to the <a href="https://my.truffleteams.com/#/settings" target="_blank">settings</a> page. You can also navigate there by clicking on your avatar/username in the left sidebar.
```

Comment:
we shouldn't use directional language

```suggestion
Once you've made sure the account you'd like to apply the coupon for exists, navigate to the <a href="https://my.truffleteams.com/#/settings" target="_blank">settings</a> page. You can also navigate there by clicking on your avatar/username in the sidebar.
```

Reply:
@eshaben why not? it provides quite a bit of context for the user